apps/web/features/error-messages/error-unexpected.tsx

```diff
@@ -0,0 +1,32 @@
+import { OctagonAlert } from "lucide-react";
+
+type Props = {
+  description?: string;
+};
+
+const ErrorUnexpected = ({ description }: Props) => {
+  return (
+    <div className="py-16 flex flex-col items-center justify-center gap-4 text-center text-neutral-800">
+      <OctagonAlert className="h-16 w-16 text-neutral-300" />
+      <p className="text-lg font-medium">Something went wrong</p>
+      <p className="text-sm text-neutral-600">
+        {description || "An unexpected error occurred. Please try again later."}
+      </p>
+    </div>
+  );
+};
+
+// Retry if we ever need it
```

Comment:
We don't need this logic in comments. This takes 5 minutes of your time to implement from scratch.

The code and requirements change frequently, and it's easy to lose sight of their value.